Source/Shaders/PolylineShadowVolumeFS.glsl

```diff
@@ -0,0 +1,81 @@
+#ifdef GL_EXT_frag_depth
+#extension GL_EXT_frag_depth : enable
+#endif
+
+varying vec4 v_startPlaneNormalEC_and_halfWidth;
+varying vec4 v_endPlaneNormalEC_and_batchId;
+varying vec4 v_rightPlaneEC; // Technically can compute distance for this here
+varying vec4 v_ecEnd_and_ecStart_X;
+varying vec4 v_texcoordNormalization_and_ecStart_YZ;
+
+#ifdef PER_INSTANCE_COLOR
+varying vec4 v_color;
+#endif
+
+void main(void)
```

Comment:
At quick glance, fragment shader does not look that expensive...very good.

Just overall fill rate will be the issue that we can address in the future.